### PR TITLE
Document changes to createImageBitmap

### PR DESCRIPTION
**Pull request by aardgoose**
> Issue number that this PR fixes (if any). For example: 'Fixes #987654321'

Adds information about support of new option parameter for rhe createBitmap() method

> What was wrong/why is this fix needed? (quick summary only)

Undocumented change

> Anything else that could help us review it

### CODE REVIEW

**Comment by Rumyra** [comment]
Thanks @aardgoose really appreciate the pr.

Can you confirm the bugzilla number please - there's a space inside the macro (see review) which means I can't find it - thank you!

File: files/en-us/mozilla/firefox/releases/93/index.html [hunk 1]
```diff
@@ -33,6 +33,10 @@ <h4 id="removals_css">Removals</h4>
 
 <h3 id="JavaScript">JavaScript</h3>
 
+<ul>
+  <li>The properties <code>imageOrientation</code> and <code>premultiplyAlpha</code> can be passed to the method {{domxref("createImageBitmap()")}} using the <code>options</code> object ({{bug(136725 1)}}).</li>
```
Comment (Rumyra) [inline files/en-us/mozilla/firefox/releases/93/index.html:37]: There's a space in the bugzilla number - can you confirm which one is reference - thanks

**Review by Rumyra — CHANGES_REQUESTED** [review]

File: files/en-us/mozilla/firefox/releases/93/index.html (hunk 1 above)
Comment (aardgoose) [reply files/en-us/mozilla/firefox/releases/93/index.html]: space removed - the bug number is now correct.

**Review by Rumyra — APPROVED** [review]

**Comment by Rumyra** [comment]
Thanks @aardgoose 👍

**Pull request merged**